### PR TITLE
[pt-br] Translate ConfigMap page to Brazilian Portuguese

### PR DESCRIPTION
**Pull request by stormqueen1990**
**Changes**
- Create the translation for the ConfigMap page to Brazilian Portuguese.
- Create glossary entries for several resources (volumes, secrets, images, configmaps).
- Updated some outdated links in glossary entries.

### CODE REVIEW

File: content/pt-br/docs/reference/glossary/service.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+aka:
+tags:
+- fundamental
+- core-object
+---
+Uma forma abstrata de expor uma aplicação que está executando em um conjunto de {{< glossary_tooltip text="Pods" term_id="pod" >}} como um serviço de rede.
+
+<!--more-->
+
+ O conjunto de Pods referenciado por um Service é (geralmente) determinado por um {{< glossary_tooltip text="selector" term_id="selector" >}}. Se mais Pods são
```
Comment (rikatz) [inline content/pt-br/docs/reference/glossary/service.md:18]: I would add in the glossary_tooltip text here "seletor" (translate the selector term)

File: content/pt-br/docs/reference/glossary/service.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ adicionados ou removidos, o conjunto de Pods que atende ao critério do selector será alterado. O Service garante que o tráfego de rede pode ser direcionado ao
```
Comment (rikatz) [inline content/pt-br/docs/reference/glossary/service.md:19]: same

File: content/pt-br/docs/reference/glossary/selector.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+title: Selector
```
Comment (rikatz) [inline content/pt-br/docs/reference/glossary/selector.md:2]: Seletor

File: content/pt-br/docs/reference/glossary/selector.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+aka: 
+tags:
+- fundamental
+---
+ Permite ao usuário filtrar uma lista de recursos com base em {{< glossary_tooltip text="labels" term_id="label" >}} (rótulos).
+
+<!--more--> 
+
+Selectors são aplicados quando um usuário faz uma pesquisa por listas de recursos para filtrá-los por labels (rótulos).
```
Comment (rikatz) [inline content/pt-br/docs/reference/glossary/selector.md:17]: Same

File: content/pt-br/docs/reference/glossary/secret.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+aka: 
+tags:
+- core-object
+- security
+---
+ Armazena dados sensíveis, como senhas, tokens OAuth e chaves SSH.
+
+<!--more--> 
+
+Permite mais controle com relação a como as informações sensíveis são armazenadas e reduz o risco de exposição acidental, incluindo [encriptação](/docs/tasks/administer-cluster/encrypt-data/#ensure-all-secrets-are-encrypted) em disco. Um {{< glossary_tooltip text="Pod" term_id="pod" >}} pode referenciar o Secret como um arquivo em um volume montado, ou o kubelet pode usar este dado quando baixa imagens para um pod. Secrets são úteis para dados confidenciais, enquanto [ConfigMaps](/docs/tasks/configure-pod-container/configure-pod-configmap/) são úteis para dados não-confidenciais.
```
Comment (rikatz) [inline content/pt-br/docs/reference/glossary/secret.md:18]: So, here maybe a "bonus" (also, would recommend to look into english translation and improve it).

There has been a discussion on pass on whether secrets values are or not encrypted (well, base64 is encoding, not encryption).

So what do you think of us taking this opportunity to:

* In this PR, add a note saying something like "Vale notar que os valores dos secrets são CODIFICADOS, mas não CRIPTOGRAFADOS em formato Base64"
* A new PR improving the English docs

File: content/pt-br/docs/reference/glossary/image.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+title: Image
```
Comment (rikatz) [inline content/pt-br/docs/reference/glossary/image.md:2]: Maybe translate here to "Imagem"?

File: content/pt-br/docs/concepts/configuration/configmap.md [hunk 7]
```diff
@@ -0,0 +1,280 @@
...
+
+Utilize um ConfigMap para manter a configuração separada do código da aplicação.
+
+Por exemplo, imagine que você esteja desenvolvendo uma aplicação que pode ser executada
+no seu computador local (para desenvolvimento) e na nuvem (para manipular tráfego real).
+Você escreve código para ler a variável de ambiente chamada `DATABASE_HOST`.
+No seu ambiente local, você configura essa variável com o valor `localhost`. Na nuvem, você
+configura essa variável para referenciar um {{< glossary_tooltip text="serviço" term_id="service" >}}
+do Kubernetes que expõe o componente do banco de dados ao seu cluster.
+Isto permite que você baixe uma imagem de contêiner que roda na nuvem e depure o exato
```
Comment (rikatz) [inline content/pt-br/docs/concepts/configuration/configmap.md:30]: s/depure o exato/depure exatamente (wdyt?)

**Review by rikatz — CHANGES_REQUESTED** [review]
Left some minor reviews here

Thanks!!

File: content/pt-br/docs/reference/glossary/service.md (hunk 1 above)
Comment (stormqueen1990) [reply content/pt-br/docs/reference/glossary/service.md]: Done!

File: content/pt-br/docs/reference/glossary/service.md (hunk 2 above)
Comment (stormqueen1990) [reply content/pt-br/docs/reference/glossary/service.md]: Done!

File: content/pt-br/docs/reference/glossary/selector.md (hunk 3 above)
Comment (stormqueen1990) [reply content/pt-br/docs/reference/glossary/selector.md]: Done!

File: content/pt-br/docs/reference/glossary/selector.md (hunk 4 above)
Comment (stormqueen1990) [reply content/pt-br/docs/reference/glossary/selector.md]: Done!

File: content/pt-br/docs/reference/glossary/image.md (hunk 6 above)
Comment (stormqueen1990) [reply content/pt-br/docs/reference/glossary/image.md]: Done!

File: content/pt-br/docs/concepts/configuration/configmap.md (hunk 7 above)
Comment (stormqueen1990) [reply content/pt-br/docs/concepts/configuration/configmap.md]: Sounds better indeed! Just updated it!

File: content/pt-br/docs/reference/glossary/secret.md (hunk 5 above)
Comment (stormqueen1990) [reply content/pt-br/docs/reference/glossary/secret.md:18]: This is an interesting point. I was working with the assumption that the parts that speak to encryption referred specifically to the possibility of encrypting data at rest (as explained on https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/), but now re-reading I can see some ambiguity here. 😅 

Should we state that, by default, secrets are only codified in base64, but can be encrypted if the cluster administrator/operator takes steps towards doing so?

File: content/pt-br/docs/concepts/configuration/configmap.md [hunk 8]
```diff
@@ -0,0 +1,280 @@
...
+        - key: "game.properties"
+          path: "game.properties"
+        - key: "user-interface.properties"
+          path: "user-interface.properties"
+```
+
+ConfigMaps não diferenciam entre propriedades com valores simples ou valores complexos,
+que ocupam várias linhas. O importante é a forma que Pods e outros objetos consomem tais valores.
+
+Para este exemplo, definir um volume e montar ele dentro do container `demo` no caminho `/config`
```
Comment (edsoncelio) [inline content/pt-br/docs/concepts/configuration/configmap.md:154]: container to contêiner

**Review by edsoncelio — COMMENTED** [review]
Congrats, i made just a small comment :)

File: content/pt-br/docs/reference/glossary/secret.md (hunk 5 above)
Comment (stormqueen1990) [reply content/pt-br/docs/reference/glossary/secret.md:18]: Opened #29985 for changing the English version.

**Comment by rikatz** [comment]
/approve
Thanks !!

**Pull request merged**